sql/core/src/main/scala/org/apache/spark/sql/execution/command/CommandUtils.scala

```diff
@@ -137,6 +136,38 @@ object CommandUtils extends Logging {
     size
   }
 
+  def calculateLocationsSizes(
```

Comment:
`calculateLocationsSizes` seems to have too many `s`; `Locations` and `Sizes`.
Maybe, `calculateLocationSize` is clear enough because the argument `paths` are `Seq`?

Reply:
How about `calculateTotalLocationSize` and `calculateSingleLocationSize` ?